mq/main/pom.xml

```diff
@@ -380,12 +380,12 @@
             <dependency>
                 <groupId>jakarta.mail</groupId>
                 <artifactId>jakarta.mail-api</artifactId>
-                <version>2.0.0-RC4</version>
+                <version>2.0.0</version>
```

Comment:
In GF [nucleus/parent/pom.xm](https://github.com/eclipse-ee4j/glassfish/blob/d6f0dfa7adf823908d95bfc3e9490ec244d509ee/nucleus/parent/pom.xml#L111)